### PR TITLE
Pre-release candidate for v2.8.0rc1

### PR DESCRIPTION
**Pull request by valeriupredoi**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,38 +20,43 @@ build:
...
+    - python >=3.8,<3.11
+    - iris >=3.4.0
     - python-stratify
     - scipy >=1.6
     # Normally installed via pip:
     - cartopy
     - cftime
-    - cf-units >=3.0.0, <3.1.0, !=3.0.1.post0
+    - cf-units
+    - compilers
```
Comment (bouweandela) [inline recipe/meta.yaml:37]: Why are compilers needed?

Comment (zklaus) [reply recipe/meta.yaml:37]: I had the same question. In #57, this works without compilers, so I am in favor of leaving it out.

Comment (valeriupredoi) [reply recipe/meta.yaml:37]: @bouweandela mentioned on Slack that `compilers` was needed for `vprof` that we no longer require, so we should be good without it, bud :+1:

**Comment by valeriupredoi** [comment]
superseded by https://github.com/conda-forge/esmvalcore-feedstock/pull/57

**Pull request closed without merging**